cli/crates/cli/tests/graphql-directive/server.rs

```diff
@@ -64,7 +64,7 @@ impl Query {
             }));
         } else if id == "3" {
             return Some(PullRequestOrIssue::Issue(Issue {
-                title: "Everythings fine".into(),
+                title: "Everything's fine".into(),
```

Comment:
nice

Reply:
This is the false fix I mentioned, it fixed it to "Everything fine" (which is ok but not exact)